.github/workflows/release.main.kts

```diff
@@ -35,9 +35,9 @@ val SIGNING_KEY by Contexts.secrets
 val SIGNING_PASSWORD by Contexts.secrets
 
 workflow(
-    name = "Publish release to Maven Central",
+    name = "Publish release to Maven Central or snapshot repo",
     on = listOf(
-        WorkflowDispatch()
```

Comment:
I would suggest keeping a manual trigger for workflow. It might be handy when you need a patch/backport in older major release. In this case, you can easily trigger publication for a different branch

Reply:
Fair point! Changed: https://github.com/krzema12/snakeyaml-engine-kmp/pull/308/commits/39076249d067fec92cd3c3089c6d6af10af3aa23